### DISCUSSION TITLE
Windows 10 volume mixer no longer shows real time audio levels

### DISCUSSION THREAD

**Discussion by Qerbas** [valinet/ExplorerPatcher#2965] [Q&A]
I've noticed this for the past few days. I can still mute and adjust system/application volume but the green bar showing the audio level doesn't work anymore. This feature is useful to me since I use it to see if volume normalization on youtube or other apps is being used or to visual check for audio issues. It's persisted through restart. The built in Windows 11 volume mixer in the system settings does work, but it's harder to read since the levels are blue on a blue background and doesn't show the levels per application. Is this an issue on the Windows side?

Explorer Patcher Version 22621.3007.63.4
Windows 11 Pro 22631.3235
Windows Feature Experience Pack 1000.22687.1000.0

**Comment by pyrates999** [comment]
Try opening up task manager and kill explorer.exe and then run task manager again and run explorer.exe.

If that doesn't work, try uninstalling the preview update you installed: https://support.microsoft.com/en-au/topic/february-29-2024-kb5034848-os-builds-22621-3235-and-22631-3235-preview-5b718126-82b1-4e14-bf01-da3500a3fc88

If that fixes it, the code to do that might have been disabled in this preview update.  If it's been disabled by removing it entirely or if it can be re-enabled using the vivetool, that would need to be looked into.

**Comment by Qerbas** [comment]
Killing explorer.exe and restarting did not resolve the issue nor did uninstalling KB5034848.

**Comment by pyrates999** [comment]
Ok, please wait for a developer to respond.
